payments/mobile-pay/other-features.md

```diff
@@ -18,6 +18,8 @@ sidebar:
 
 ## Payment Resource
 
+{% include payment-resource.md  payment-instrument="mobilepay" showStatusOperations='false' %}
```

Comment:
I'm not 100% certain, but I think `false` needs to be passed as a literal and not as a quoted string.

```suggestion
{% include payment-resource.md  payment-instrument="mobilepay" showStatusOperations=false %}
```